packages/react/src/components/Provider/Provider.tsx

```diff
@@ -146,41 +160,49 @@ class Provider extends React.Component<WithAsProp<ProviderProps>> {
   }
 
   render() {
-    const { as, theme, variables, children, ...unhandledProps } = this.props
-
+    const {
+      as,
+      theme,
+      rtl,
+      disableAnimations,
+      renderer,
+      variables,
+      children,
+      ...unhandledProps
+    } = this.props
+    const context: ProviderContextInput = {
```

Comment:
nit: just for the sake of better explicitness, may suggest to rename to `inputContext` or `contextFromProps`